src/services/sync-checker.ts

```diff
@@ -1,13 +1,14 @@
 import { Configuration, HTTPMethod, Node, Pocket, PocketAAT, RelayResponse } from '@pokt-network/pocket-js'
 import { MetricsRecorder } from '../services/metrics-recorder'
 import { Redis } from 'ioredis'
-import { blockHexToDecimal, checkEnforcementJSON, getNodeNetworkData } from '../utils'
+import { removeNodeFromSession, getNodeNetworkData } from '../utils/cache'
+import { checkEnforcementJSON } from '../utils/string'
+import { blockHexToDecimal } from '../utils/block'
 
 const logger = require('../services/logger')
 
 import axios from 'axios'
```

Comment:
Let's move this up with the libray imports. Let's try to always use this order:

- `node_modules` imports (`axios`, `ioredis`, etc...)
- `node_modules` imports with a `@`
- relative imports.
- `require` imports because they're ugly.